DependencyInjection/DoctrineExtension.php

```diff
@@ -59,6 +59,10 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = $this->getConfiguration($configs, $container);
         $config = $this->processConfiguration($configuration, $configs);
 
+        if (method_exists($container, 'resolveEnvPlaceholders')) {
+            $config = $container->resolveEnvPlaceholders($config, true);
```

Comment:
resolving env placeholders in the whole config is a no-go, as this would mean that DB credentials are not read at runtime anymore for instance (requiring a cache clearing after changing the env variable, while using `%env()%` is precisely meant to avoid that)